gen/OpenIddict.Client.WebIntegration.Generators/OpenIddictClientWebIntegrationGenerator.cs

```diff
@@ -699,10 +699,12 @@ public sealed partial class {{ provider.name }}
         {{~ end ~}}
         public {{ provider.name }} Set{{ setting.property_name }}({{ setting.clr_type }} {{ setting.parameter_name }})
         {
+            {{~ if setting.clr_type != 'bool' ~}}
```

Comment:
Instead of making the `if (...)` block conditional, could you please copy the entire method and put it in its own `{{~ else if setting.clr_type == 'bool' ~}}` / `{{~ end ~}}`?

Reply:
Hopes I have understood what you meant. I will try it tomorrow. 🌃

Reply:
Thanks! Have a great night 🌔